package.json

```diff
@@ -13,5 +13,8 @@
         "1.1.0": "provideLinter"
       }
     }
+  },
+  "dependencies": {
+    "atom-linter": "^3.2.0"
```

Comment:
I know, and you have the correct one. This is just an npm helper module we've made. You can find the repo at github.com/AtomLinter/atom-linter